<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign In</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #FFFFFF;
    }
    QLabel {
        font-family: &quot;Inter&quot;;
        color: #333333;
    }
    QLineEdit {
        border: 1px solid #E0E0E0;
        border-radius: 8px;
        padding: 12px;
        font-family: &quot;Inter&quot;;
        font-size: 14px;
        color: #333333;
        background-color: #F8F8F8;
    }
    QLineEdit:focus {
        border: 1px solid #4285F4;
    }
    QPushButton {
        border-radius: 8px;
        padding: 12px;
        font-family: &quot;Inter&quot;;
        font-size: 16px;
        font-weight: bold;
    }
    #loginButton { /* Specific style for the Sign In button */
        background-color: #4285F4; /* Google Blue */
        color: #FFFFFF;
        border: none;
    }
    #loginButton:hover {
        background-color: #357AE8;
    }
    #forgotPasswordLabel, #signUpLabel {
        color: #4285F4;
        text-decoration: none; /* remove underline for QLabels acting as links */
    }
    #signUpContainer QLabel {
        color: #666666; /* For &quot;Don't have an account?&quot; */
    }
    #iconLabel {
        font-size: 48px; /* Larger icon */
        color: #4285F4;
        /* You might replace this with an actual SVG/image later */
    }
    #titleLabel {
        font-size: 28px;
        font-weight: bold;
        margin-top: 10px;
        margin-bottom: 20px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignCenter">
    <widget class="QLabel" name="iconLabel">
     <property name="text">
      <string>&#x2022;&#x2022;&#x2022;</string>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignCenter">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Sign in</string>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="placeholderText">
      <string>E-mail Address *</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumHeight">
      <number>48</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password *</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumHeight">
      <number>48</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QLabel" name="forgotPasswordLabel">
     <property name="text">
      <string>&lt;a href=&quot;#&quot; style=&quot;text-decoration:none; color:#4285F4;&quot;&gt;Forgot password?&lt;/a&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Sign in</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumHeight">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_middle">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignCenter">
    <widget class="QWidget" name="signUpContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_signUp">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="noAccountLabel">
        <property name="text">
         <string>Don't have an account?</string>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="signUpLabel">
        <property name="text">
         <string>&lt;a href=&quot;#&quot; style=&quot;text-decoration:none; color:#4285F4; font-weight:bold;&quot;&gt;Sign Up!&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
